checks/normandy/uptake_error_rate.py

```diff
@@ -11,13 +11,15 @@
 from typing import Dict, List, Tuple
 
 from poucave.typings import CheckResult
-from poucave.utils import fetch_redash
+from poucave.utils import fetch_json, fetch_redash
 
 
 EXPOSED_PARAMETERS = ["max_error_percentage", "min_total_events"]
 
 REDASH_QUERY_ID = 67658
 
+NORMANDY_URL = "{server}/api/v1/recipe/signed/?enabled=1"
```

Comment:
At some point we need to get Poucave to use the v3 API or GraphQL. I'd like to avoid using v1 much longer. That being said, this kind of "get all enabled recipes" isn't super easy in the v3 API, so I'm not going to insist you change it here.